### PR TITLE
Yessenia Rodriguez string-manipulation-1

### PR DESCRIPTION
**Pull request by yesseniarod**
Screen Shot 2021-01-29 at 4 26 53 PM

### CODE REVIEW

File: string-manipulation-1/src/capitalize.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+function capitalize(word) {
+  var firstLetter = word[0];
+  var firstLetterUpperCase = firstLetter.toUpperCase();
+  var cutWord = word.slice(1);
+  var lowerCaseAll = cutWord.toLowerCase();
+  var updatedWord = firstLetterUpperCase + lowerCaseAll;
+  var i = 0;
+  while (i < word.length) {
+    i++;
+  }
```
Comment (thebearingedge) [inline string-manipulation-1/src/capitalize.js:11]: You can remove this whole chunk 😄

File: string-manipulation-1/src/get-first-char.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* exported getFirstChar */
+function getFirstChar(string) {
+  var newString = '';
+  for (var i = 0; i < string.length; i++) {
+    newString = string[0];
+  }
```
Comment (thebearingedge) [inline string-manipulation-1/src/get-first-char.js:6]: You can grab `string[0]` without the loop.

File: string-manipulation-1/src/get-last-char.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/* exported getLastChar */
+function getLastChar(string) {
+  var lastString = '';
+  var lastIndex = '';
+  for (var i = 0; i < string.length; i++) {
+    lastString = string.length - 1;
+    lastIndex = string[lastString];
+  }
```
Comment (thebearingedge) [inline string-manipulation-1/src/get-last-char.js:8]: You can grab the `string[string.length - 1]` without the loop.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Nice work so far, please see my notes.

File: string-manipulation-1/src/get-last-char.js (hunk 3 above)
Comment (yesseniarod) [reply string-manipulation-1/src/get-last-char.js]: Screen Shot 2021-01-29 at 4 56 09 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**